modules/engines/engine-hdb/src/main/java/com/sap/xsk/hdb/ds/model/hdbtable/XSKDataStructureHDBTableCalculatedColumnModel.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2022 SAP SE or an SAP affiliate company and XSK contributors
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and XSK contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.sap.xsk.hdb.ds.model.hdbtable;
+
+public class XSKDataStructureHDBTableCalculatedColumnModel extends XSKDataStructureHDBTableCommonColumnModel {
+
+  private String columnName;
+
+  private String statement;
+
+  public XSKDataStructureHDBTableCalculatedColumnModel() {
+  }
+
+  public XSKDataStructureHDBTableCalculatedColumnModel(String columnName, String statement) {
+    this.columnName = columnName;
+    this.statement = statement;
+  }
+
+  public String getColumnName() {
+    return columnName;
+  }
+
+  public void setColumnName(String columnName) {
```

Comment:
The setters are used inside the HdbddTransformer when parsing HDBDDs, due to the current logic, refactoring is needed, in order to remove them and even then, some setters are used in multiple places. This refactoring will not bring any benefits in my opinion.

Reply:
I agree, in addition, this is old logic that is no longer necessary. Since we are not using alter statements anymore, there is no need to add new models and more logic. Everything is changed in [9efed80](https://github.com/SAP/xsk/pull/1716/commits/9efed803d931c5dc2fc9a840b06615d0e11f32a3)